src/routes/safe/components/Apps/confirmTransactions.js

```diff
@@ -38,9 +49,35 @@ const confirmTransactions = (
   closeModal: () => void,
   onConfirm: () => void,
 ) => {
+  const isTxValid = (t) => {
```

Comment:
move this functions outside the component, it doesn't depend on any props

Reply:
done!